internal/pkg/db/mongo/models/command.go

```diff
@@ -88,10 +88,12 @@ func (c *Command) FromContract(from contract.Command) error {
 	}
 
 	c.Name = from.Name
+	c.Get = &Get{}
 	err := c.Get.FromContract(*from.Get)
```

Comment:
`*from.Get`

We're eventually going to get rid of these pointers, right? This is just a holdover from the existing code and is pending @jdharms work on removing the MemoryDB, correct?

Reply:
Correct.